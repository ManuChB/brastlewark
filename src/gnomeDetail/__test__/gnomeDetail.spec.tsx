import React from 'react';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import GnomeDetails from '../gnomeDetail';

configure({
    adapter: new Adapter()
});


describe('Gnome detail component', () => {
    const gnomeDetails = {
        gnome: {
            id: 1,
            name: 'name',
            thumbnail: 'image',
            age: 100,
            weight: 100,
            height: 100,
            hair_color: 'red',
            professions: [],
            friends: [],
            onPress: jest.fn()
        },
        friendsDetails: [],
        onFriendPress: jest.fn()
    };

    it('renders as expected', () => {
        //when
        const wrapper = shallow(<GnomeDetails {...gnomeDetails} ></GnomeDetails>);
        //expect
        expect(wrapper).toMatchSnapshot();
    });
});