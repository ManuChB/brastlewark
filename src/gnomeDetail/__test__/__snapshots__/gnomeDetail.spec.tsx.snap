// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Gnome detail component renders as expected 1`] = `ShallowWrapper {}`;
